lucas_scheinkerman/README.md

```diff
@@ -1 +1,45 @@
+
+
+-Besides the angular velocity of the wheels, there are two more factors that define the pose variation (position and direction) of the robot. These are the **wheels radius** and the **distance between the two wheels**. <br>
+Intuitively, its easy to see that the longer the distance between the wheels, greater angular velocities of the wheels will be needed for getting the same velocity of the robot as with shorter wheel distance. This is because as the wheels distance becomes larger, the circumference which will be involved in the rotational velocity of the robot will be bigger, which makes the rotational velocity of the robot lower.  <br>
+With the wheels radius the opposite effect takes place. As the radius becomes bigger, with constant wheel angular velocity, the velocity of the robot increases. 
+
+
+![Reference graphic](media/reference.png)
+
+- Mathematically, all that we explained before can be described with the following equations:
+    - V_robot = (Vr + Vl)/2
+    - V_x = V_robot * cos(theta)
+    - V_y = V_robot * sin(theta)
+    - w_robot = (r/L)*(Vr - Vl)        <br>
+    ('L' being the distance between the wheels, 'r' the radius of the wheels and 'theta' the angle of orientation of the robot)
+
+- From these equations, we can rearrange them such that we can define both wheels velocities as function of the linear and angular velocity desired for the robot. We'll want to do that because the system is moved controlling the wheel velocities, but the real parameters that we want to control are the linear and angular velocities _of the robot_. After rearranging, the equations are:
+    - V_r = (V_robot + (L/2)w_robot)
+    - V_l = (V_robot - (L/2)w_robot)
+
+- Finally, there are a few physical considerations that can introduce error to the equations described before, or can make the model for movement more complex. These simplifications are:
+    - Both wheels velocities are constant when they are present. That is, there is no transient state in the velocity of the wheels. 
+    - The wheels do not slip
+    - The surface is flat
+    - The wheels have a fixed position, and can't move in any way except for their angular velocity. (As opposed to the case of, for example, a mecanum wheel)
+
+Ignoring these considerations helps us find a simple model for movement, but reduces the cases in which we can use the equations described above. So it's important to know whether we can apply these equations or not every time we're considering using a DDR.
+
+
+
+
+###### References:
```

Comment:
Use the proper header indentation.

Reply:
Keep in mind that most of your commits say "Update README". Try to be more specific on what you did when you work with code. It'll help you to debug code and delete something if necessary.